PFS-IPMU-00001/maintenance.rst

```diff
@@ -0,0 +1,140 @@
+Configurations and maintenance
+======
+
+ToC
+
+* `LDAP account`_
+* `LDAP server`_
+* `JIRA`_
+* `Special cases on replacing host`_
+* `Let's encrypt / certbot`_
+* `Updating web services`_
+* `Mailman`_
+
+LDAP account
+------
+
+Use 
+`LDIF generation script <https://github.com/Subaru-PFS/pfs_infra_ipmu.git>`_ 
+for operation (`ldap-admin` in repository). 
+UID number registration (listing) file is required for new account generation, 
+use one at `/server/admin/github-scripts/pfs_infra_ipmu/ldap-admin/`. 
+
+* Create target list file (see command help below)
+* Execute `create-ldif.pl` script
+* Execute `.cmd` file generated to run ldapadd/ldapmodify
```

Comment:
Maybe the step to add jpeg photo, if any, is missing? 
(from recent work)

Reply:
ah, yes.
or do we better to modify our procedure not to require photo on account creation, but ask them to add on an initial configuration change?

Reply:
Just for recording, Atsushi and I talked about this. Atsushi will modify `create-ldif.pl` to have an option for adding photo.

Reply:
note added.